pkg/controllers/awsloadbalancercontroller/rbac_rolebinding.go

```diff
@@ -98,8 +98,8 @@ func (r *AWSLoadBalancerControllerReconciler) currentRoleBinding(ctx context.Con
 	return true, obj, nil
 }
 
-func desiredRoleBinding(ctx context.Context, namespace string, sa *corev1.ServiceAccount) *rbacv1.RoleBinding {
-	return buildRoleBinding(commonResourceName, namespace, commonResourceName, sa)
+func desiredRoleBinding(ctx context.Context, name, namespace string, sa *corev1.ServiceAccount) *rbacv1.RoleBinding {
```

Comment:
I feel like we should be getting the role name and the service account name from their individual ensure functions and passing them into this function, rather than relying on convention. But we can do that in a followup PR.